content/blog/2018/announcing-1.0/index.md

```diff
@@ -0,0 +1,68 @@
+
+### Upgrading Istio versions
+
+Istio is built on the sidecar model – every pod in the mesh has an Envoy sidecar proxy running right alongside it to add the service mesh smarts. All traffic to and from the application container goes through the sidecar first. Previous versions of Istio let you inject sidecar manually or at the time that the kubernetes deployment was created. The drawback of deployment-time injection comes when you want to upgrade your service mesh to run a newer version of the sidecar.  You either have to patch the deployments in your cluster, or delete and then recreate them.
+
+With 1.0, you can use a MutatingWebhook Pod Admission Controller. Whenever Kubernetes is about to create a pod, it lets its pod admission controllers take a look and decide to allow, reject or allow-with-changes that pod. Every admission controller gets a shot.  Istio provides one of these admission controllers that injects the current sidecar and init container. This make it easier to upgrade the service mesh.
+
+### Pilot scalability
+
+One of the issues in the earlier version of Istio was degrading pilot performance as number of service grows. When you make changes in the cluster like add/remove services, add/remove pods etc., each time Istio has to apply new pilot configurations. On top of that, each sidecar had to pull the changes from the pilot. This pull model slow down the performance if you have to do it for lot of services. The new v3 config APIs were added to pilot using the new Envoy v2 config APIs which uses a push model instead of pull. This change improved a performance dramatically compared to earlier version.
+
+### Better external service support
+
+In the earlier version of Istio, if you wanted a service in the mesh to communicate with a TLS service outside the mesh, you had to modify your service to speak http over port 443, so that istio could route it correctly. Now, that Istio can use SNI to route traffic, you can leave your service alone, and configure Istio to allow it to communicate with that external service by hostname. Since this is TLS passthrough, you don’t get L7 visibility of the egress traffic, but since you don’t need to modify your service, it allows you to add services to the mesh that you might not have been able to before.
+
```

Comment:
This is all related to networking. Nothing from policy/telemetry, etc. ?
Also, do we really need this content? The blog was short and sweet till line 27

Reply:
agreed... it feels random feature being called out into more details.